snc-library.sh

```diff
@@ -56,12 +57,17 @@ function run_preflight_checks() {
 
         echo "Checking libvirt and DNS configuration"
 
-        LIBVIRT_URI=qemu+tcp://localhost/system
+	LIBVIRT_URI=qemu:///system
+	if [ ${preset} == "snc" ]; then
```

Comment:
We have `BUNDLE_TYPE` which can be `snc`, `okd`, `microshift`, but here we have `preset` which defaults to `snc`, so `preset == snc` covers both `snc` and `okd` `BUNDLE_TYPE`. This is *very* confusing, can this be done slightly differently?